src/utils/validation.js

```diff
@@ -0,0 +1,32 @@
+export const validationFunction = {
+  userName: username => {
+    if (username === '') {
+      return false;
+    } else {
+      return true;
+    }
+  },
+
+  userId: birth => {
+    if (birth.length === 8) {
+      return true;
+    } else {
+      return false;
+    }
+  },
+  userBirth: number => {
+    if (number.length === 11) {
+      return true;
+    } else {
+      return false;
+    }
+  },
+  userNum: id => {
+    var reg_id = /^[a-zA-z0-9]{4,12}$/;
+    return reg_id.test(id) ? true : false;
```

Comment:
삼항 쓸 필요 없습니다 애초에 test가 true or false를 리턴합니다
```suggestion
    return reg_id.test(id)
```

Reply:
수정하였습니다!

Reply:
수정했습니다!